hw1/task1/get_stats.py

```diff
@@ -0,0 +1,140 @@
+import pandas as pd
+import numpy as np
+
+
+def convert_type_function(type_to_convert, return_if_exception=None):
+    def return_function(x):
+        try:
+            x = type_to_convert(x)
+        except ValueError:
+            return return_if_exception
+        else:
+            return x
+    return return_function
+
+
+def read_value_data(path):
```

Comment:
can you add validation to check that 'lpep_pickup_datetime' is always equals or less than 'lpep_dropoff_datetime'?